ci-operator/step-registry/ipi/conf/vsphere/staticip/verify/ipi-conf-vsphere-staticip-verify-commands.sh

```diff
@@ -38,7 +45,8 @@ oc project openshift-machine-api
 # start at the 24th ip address entry to generate a /29 address pool
 address_cidr=$(jq -r --arg PRH "$primaryrouterhostname" --arg VLANID "$vlanid" '.[$PRH][$VLANID].ipAddresses[24]' "${SUBNETS_CONFIG}")
 
-curl -k https://raw.githubusercontent.com/rvanderp3/machine-ipam-controller/main/hack/ci-resources.yaml | oc process -p GATEWAY="${gateway}" -p PREFIX="${cidr}" -p ADDRESS_CIDR="${address_cidr}/29" --local=true -f - | oc create -f -
+# We are ignoring unknown parameters for now due to sync between repo and ci
+curl -k https://raw.githubusercontent.com/openshift-splat-team/machine-ipam-controller/main/hack/ci-resources.yaml | oc process -p GATEWAY="${gateway}" -p PREFIX="${cidr}" -p ADDRESS_CIDR="${address_cidr}/29" -p IPAM_VERSION="${image_version}" --local=true --ignore-unknown-parameters=true -f - | oc create -f -
```

Comment:
Its always a `/29`?

Reply:
we've been treating the static ip range to be 29 since we only need 2 IPs and make sure they do not conflict. I have not changed that part w/ this PR.